plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceRuntimeInfo.java

```diff
@@ -355,4 +378,18 @@ public String projectsRoot() {
 
         return serversConf;
     }
+
+    /**
+     * Checks environment variable {@code CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS} to determine if
+     * direct addresses of containers should be used.
+     * @return true if {@code CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS} is "true", false otherwise.
+     */
+    protected boolean useInternalContainerAddresses() {
+        String useInternalContainerAddresses = System.getenv(CHE_DOCKER_USE_INTERNAL_CONTAINER_ADDRESS);
```

Comment:
@garagatyi Thanks for pointing me to that PR -- I've renamed the property to `che.docker.ip.use_internal_address` for readability. This has the side effect of not being able to override the property with an environment variable until your change is merged.

I've modified `LocalDockerInstanceRuntimeInfo` to not use environment variables anymore and updated the docs. Previously, the properties `che.docker.ip` and `che.docker.ip.external` were overridden through both the environment variable `CHE_DOCKER_IP`/`CHE_DOCKER_IP_EXTERNAL` but also through `CHE_DOCKER_MACHINE_HOST` and `CHE_DOCKER_MACHINE_HOST_EXTERNAL`. With the change only the former has an effect.